### PR TITLE
Updated Russian translation

### PR DESCRIPTION
**Pull request by Rinat84**
Some places did not translate, did not find where they are and what they are doing, they are probably for WSL distribution

### CODE REVIEW

**Comment by mintty** [comment]
> Some places did not translate, did not find where they are and what they are doing

You mean you did not know how to translate some messages because their purpose was unclear?
Please list which ones, so I can improve their description.

**Comment by Rinat84** [comment]
Here are the lines.
msgid "HTML Screen Dump"
msgid "Clear Scrollback"
msgid "Send Break"

Do I think this is how the browser tabs?
msgid "Session switcher"
msgid "Session launcher"
then it will not be necessary to correct.

and this line does not work translation
msgid "Leading: %d, Bold: %s, Underline: %s"

**Comment by Rinat84** [comment]
I don’t know how to add a commit that referred to this request
so i will write

msgid "Session switcher"
msgstr "Параллельные сеансы"

msgid "Session launcher"
msgstr "Новый сеанс"

msgid "&Log to File"
msgstr "Сохранять логи в файл"

msgid "HTML Screen Dump"
msgstr "Сохранить снимок экрана в HTML"

msgid "Clear Scrollback"
msgstr "Сброс полосы прокрутки"

msgid "Send Break"
msgstr "Прервать текущую задачу"

**Comment by mintty** [comment]
How does the entry msgid "Leading: %d, Bold: %s, Underline: %s" not work? It does not have a translation in ru.po yet.

**Comment by BrianInglis** [comment]
For the technical term Leading, it is unclear as to whether it means Line Spacing or Line Padding, or what option sets it to what value, and Bold and Underline argument alternatives and their msgids are unclear, unspecified, or unavailable? Msgids exist for Font, font, Colour, and manual but not colour or both, nor other Bold and Underline settings such as special background. Commenting relevant options and argument values in the message.pot file would make translation easier or possible.

**Comment by Rinat84** [comment]
Found why it was not translated, my mistake was.

msgid "Leading: %d, Bold: %s, Underline: %s"
msgstr "Интервал: %d, стиль1: %s, стиль2: %s"

**Comment by mintty** [comment]
While I don't speak Russian, the double occurrence of "стиль" only distinguished by 1 or 2 does not match to Bold vs. Underline for me. Please recheck with the newly enhanced message description:

#. __ Options - Text: font properties information: "Leading": total line padding (see option RowSpacing), Bold/Underline modes (font or manual, see options BoldAsFont/UnderlineManual/UnderlineColour)

Does that sound any clearer?

**Comment by Rinat84** [comment]
if literally translate it will be long phrases
msgstr "Интервал: %d, жирный: %s, подчеркнутый: %s"

**Comment by mintty** [comment]
That is in fact too long for the menu; can you make the last one an abbreviation?

**Comment by BrianInglis** [comment]
Do you have a msgid for "colour" or are you relying on lowercasing in languages' scripts where that is supported? 
For next time you consider layout: https://www.w3.org/International/articles/article-text-size 
TL;DR: Korean, English, and Chinese are dense: allow 30% extra space for long lines, 3x the space for abbreviations or words. 
Layout using German or Italian words and phrases, without abbreviations, and round up an em space.

**Comment by mintty** [comment]
There is no separate "colour" string in the UI.
The UI can be scaled already as described in https://github.com/mintty/mintty/wiki/Tips#adding-translations-to-localization using msgid "100".

**Comment by Rinat84** [comment]
if you remove spaces
msgstr "Интервал:%d, жирный:%s, андерскор:%s"
